tests/manage/mcg/ui/test_mcg_ui.py

```diff
@@ -281,6 +281,12 @@ def test_obc_creation_and_deletion(
         )
 
         obc_ui_obj = ObcUI(setup_ui_class)
+
+        if (
+            config.DEPLOYMENT["external_mode"]
+            and storageclass == "ocs-storagecluster-ceph-rgw"
+        ):
+            storageclass = "ocs-external-storagecluster-ceph-rgw"
```

Comment:
I think it would be nicer to have it as one of the constants in ocs_ci/ocs/constants.py

Reply:
fixed